.github/workflows/build.yaml

```diff
@@ -17,4 +19,9 @@ jobs:
       with:
         node-version: ${{ matrix.node-version }}
     - run: npm install
-    - run: npm run build
\ No newline at end of file
+    - run: npm update
+    - run: npm run build
+    - uses: stefanzweifel/git-auto-commit-action@v4
+      with:
+        skip_dirty_check: false
+        commit_message: 'fix(items): new items'
```

Comment:
Do you need the rest of perks of same module?
Like commit username/email/name
https://github.com/stefanzweifel/git-auto-commit-action#usage

Reply:
the dirty check is pretty useful. we can have it submit them as the drone bot, but probably don't need to. i don't mind pulling this in since it abstracts out the rest of the git logic, in case it ever changes